core/data/core.joke

```diff
@@ -3582,6 +3582,11 @@
   (^Map [^Associative m k ^Callable f x y z & more]
    (assoc m k (apply f (get m k) x y z more))))
 
+(defmacro type?
```

Comment:
I think this macro is not very useful by itself and it's better not to expand core's surface area (it's already huge) and diverge from Clojure without necessity. So I think it probably makes sense to inline this into `doc` macro.